### ISSUE TITLE
HamburgerMenu is either cut off or scrolls past on the bottom

### ISSUE THREAD

**Issue by waikoo** [waikoo/thrifty#2]
**Problem**:
If I have the useEffect on in `app/components/navigation/HamburgerMenu.tsx`, if you visit the [live site](https://thrifty-seven.vercel.app/en/women) on a mobile device and try to scroll down within the hamburger menu that appears, it's cut off.

If you visit the site from a PC and use the device toolbar with the 320px preset to imitate a mobile view, it appears as scrollable, and won't scroll past its container:
![as shown here](https://github.com/waikoo/thrifty/assets/74474701/eec912dd-ea78-4fa7-bee7-80786b0c5828)

If you uncomment the useEffect mentioned earlier, it'll scroll past its container:
![as shown here](https://github.com/waikoo/thrifty/assets/74474701/61cc0ec8-6c06-472b-acf6-7db812a6dee7)

**Goal**:
Have the same behavior on a mobile device as in the device toolbar -> a **scrollable hamburger menu** that **doesn't** scroll past the vertical bottom of its container on the live site visited from a **mobile device**.

**Question**:
Why does this have such different behavior on a mobile device as opposed to the device toolbar in dev tools?

**Comment by LDraco-Sharif** [comment]
Hi @waikoo, I looked into the code and it seems that "h-screen" is not behaving well on mobile devices. I have made a fix with [this](https://github.com/waikoo/thrifty/pull/3) PR, based on responses from [this](https://stackoverflow.com/questions/61308575/tailwind-h-screen-doesn-t-work-properly-on-mobile-devices) question. I tested it on my android device. So, let me know if this fix is working on your end.

Note: Since, I am not a member of the Team, I cannot authorize Vercel and thus, Vercel authorization is failing on the PR.
